spec-main/version-bump-spec.ts

```diff
@@ -1,7 +1,14 @@
 import { expect } from 'chai';
-import { nextVersion } from '../script/release/version-bumper';
+import { nextVersion, shouldUpdateSupported, updateSupported } from '../script/release/version-bumper';
 import * as utils from '../script/release/version-utils';
 import { ifdescribe } from './spec-helpers';
+const fs = require('fs');
+const path = require('path');
+const { promisify } = require('util');
```

Comment:
the fs module has built-in promises already - let's use those instead of promisifying?

```js
const { promises: fs } = require('fs');
```

Reply:
Definitely! Updated in both the script and the spec, thanks for the catch 🙇‍♀️